mne/decoding/tests/test_ems.py

```diff
@@ -21,6 +21,15 @@
 event_id = dict(aud_l=1, vis_l=3)
 
 
+@requires_sklearn
+def test_ems_params():
```

Comment:
Style complains that all of these `test_*` functions are public and need a docstring -- see existing `test_*` functions for examples